chamber/models/fields.py

```diff
@@ -194,3 +196,44 @@ class EnumSequencePositiveIntegerField(EnumSequenceFieldMixin, models.PositiveIn
 
 class EnumSequenceCharField(EnumSequenceFieldMixin, models.CharField):
     pass
+
+
+class PriceField(DecimalField):
+
+    def __init__(self, *args, **kwargs):
+        self.currency = kwargs.pop('currency', ugettext('CZK'))
+        custom_kwargs = kwargs.copy()
+        custom_kwargs.update({
+            'decimal_places': 2,
+            'max_digits': 10,
```

Comment:
Souhlasím s Lubošem, někdy můžeme chtít větši přesnost... proč to vlastně kopíruješ do custom kwargs? Prostě bych do kwargs dal max_digits 10, pokud tam nebudou a předal bych kwargs dál.